src/coq/constants/proputils.ml

```diff
@@ -0,0 +1,34 @@
+(*
+ * Utilities for Prop oriented types and tactics
+ *)
+
+open Constr
+open Names
+open Apputils
+
+let coq_init_logic =
+  ModPath.MPfile
+    (DirPath.make (List.map Id.of_string ["Logic"; "Init"; "Coq"]))
+
+(* --- Constants --- *)
+
+(* Logical or *)
+let logical_or : types =
+  mkInd (MutInd.make1 (KerName.make2 coq_init_logic (Label.make "or")), 0)
+  
+(* left constructor of \/ *)
+let or_introl : types =
+  mkConstruct (fst (destInd logical_or), 1)
+
+(* right constructor of \/ *)
+let or_intror : types = 
+  mkConstruct (fst (destInd or), 2)
```

Comment:
`logical_or`?

Reply:
Yes that's right. Thank you I forgot to change it when I realized "or" is a reserved keyword.

Reply:
Let me know when you've updated this so I can approve it